lambda/README.md

```diff
@@ -8,18 +8,17 @@ zip lambda.zip main
 
 1. Start localstack. If you're on a Mac, ensure you are running the docker daemon.
 ```
-SERVICES=lambda,es LAMBDA_EXECUTOR=docker localstack start
+SERVICES=lambda,es,sns LAMBDA_EXECUTOR=docker localstack start
 ```
 
 1. Upload zip and create a function definition
 ```
-AWS_ACCESS_KEY_ID=999999 AWS_SECRET_ACCESS_KEY=1231 aws \
+AWS_REGION=localstack AWS_ACCESS_KEY_ID=999999 AWS_SECRET_ACCESS_KEY=1231 aws \
 --endpoint-url http://localhost:4574 lambda create-function \
 --function-name f1 \
 --runtime go1.x \
 --role r1 \
 --handler main \
---environment "Variables={ES_HOST=192.168.2.7}" \
```

Comment:
Why is this removed?

Reply:
I think I'm behind when you added that. Will get it back in, thanks for the heads up.